query/src/main/scala/filodb/query/exec/ExecPlan.scala

```diff
@@ -221,11 +221,22 @@ trait ExecPlan extends QueryCommand {
                   MeasurementUnit.time.milliseconds)
               .withTag("plan", getClass.getSimpleName)
               .record(Math.max(0, System.currentTimeMillis - startExecute))
+            // Does not include sizes of RV's that arrived at this node as SerializableRangeVectors.
             val numDataBytes = builder.allContainers.map(_.numBytes).sum
             val numKeyBytes = r.foldLeft(0)(_ + _.key.keySize)
             val resultSize = numDataBytes + numKeyBytes
             SerializedRangeVector.queryResultBytes.record(resultSize)
-            querySession.queryStats.getResultBytesCounter(Nil).addAndGet(resultSize)
+            // Includes sizes of all RV's in this subtree.
+            val totalSize = querySession.queryStats.getResultBytesCounter(Nil).addAndGet(resultSize)
```

Comment:
I've made the change, but I'm still skeptical that `resultSize` has the count we want, since (1) `numDataBytes` is initialized like this:
```
val numDataBytes = builder.allContainers.map(_.numBytes).sum
```
and (2) `builder` is only mutated by the `RangeVector` case (not `SerializableRangeVector`) of an earlier `map` over the RV's of the `ExecResult`.

For example, `numDataBytes` would undercount for `LocalPartitionDistConcatExec`  because its `ExecResult` plans would (in most cases) be `SerializableRangeVectors`. I can demonstrate this undercount locally.